BookingsApi/BookingsApi.DAL/Migrations/20220314104958_FixCriminalInjuiriesCompensationCaseType.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class FixCriminalInjuiriesCompensationCaseType : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("update [dbo].[HearingRole] set [UserRoleId] = 7 where [Id] = 989 and [CaseRoleId] = 311");
```

Comment:
Don't think we need this:

`and [CaseRoleId] = 311`

as the Id is already unique.

Reply:
yes, it is not required but being there is no harm.